### PR TITLE
Update vimgolf.el

### PR DESCRIPTION
**Pull request by riscy**
A couple package-lints re: https://github.com/melpa/melpa/pull/6599

### CODE REVIEW

File: vimgolf.el [hunk 1]
```diff
@@ -446,8 +447,7 @@ argument is dropped on the floor."
   ;; TODO is (with-current-buffer (current-buffer) …) necessary here?
   (with-current-buffer (current-buffer)
     (let ((html (vimgolf-parse-html-entites
-                 (replace-regexp-in-string "\n" "" (buffer-string))))
-          (start 0))
```
Comment (riscy) [inline vimgolf.el:450]: This `start` appeared to be unused -- sanity check for me.

File: vimgolf.el [hunk 2]
```diff
@@ -44,6 +45,8 @@
 ;;; Code:
 
 (require 'json)
+(require 'url-http)
+(defvar url-http-end-of-headers)
```
Comment (riscy) [inline vimgolf.el:49]: This'll resolve the `reference to free variable` warning.

**Comment by timvisher** [comment]
LGTM. Thanks!

**Comment by timvisher** [comment]
https://github.com/timvisher/vimgolf.el/commit/8558028d803914379d88f3a39cefdb7826b1fb89

**Pull request closed without merging**